requirements_gui.txt

```diff
@@ -1 +1 @@
-PySide2==5.15.2
+PySide2==5.11.2
```

Comment:
I don't think downgrading PySide is something we want to do. Is there a change in here that requires it? Or did you have some trouble installing `5.15.2` in your environment?

Reply:
Yeah sorry, 5.15.2 does not work on my environment for some reason and I forgot to exclude it from my commit. I changed it back in the most recent update. This should be ready to merge now.